light-system-programs/tests/functional_tests.ts

```diff
@@ -43,8 +40,7 @@ import { bs58 } from "@coral-xyz/anchor/dist/cjs/utils/bytes";
 var POSEIDON;
 var RELAYER;
 var KEYPAIR;
-var deposit_utxo1: Utxo;
-var TRANSACTION_NONCE = 0;
+// var TRANSACTION_NONCE = 0;
```

Comment:
why are we commenting the variable?

Reply:
in the cleanup, I didn't remove unused variables I wasn't able to address (code's not mine) so I just commented them

Reply:
feel free to remove, we used this as domain separation for viewing keys but changed it. It's not used anymore.

Reply:
removed ✅